src/emqx_acl_cache.erl

```diff
@@ -47,6 +48,7 @@
 %% Wrappers for key and value
 cache_k(PubSub, Topic)-> {PubSub, Topic}.
 cache_v(AclResult)-> {AclResult, time_now()}.
+discard_k() -> {?MODULE, discard_stamp}.
```

Comment:
use a timestamp here ?

Reply:
could you elaborate more?

Reply:
sorry, brainfart. this is the key, which does not include the value (timestamp)